apps/etterlatte-migrering/src/main/kotlin/migrering/verifisering/Verifiserer.kt

```diff
@@ -151,10 +155,16 @@ internal class Verifiserer(
             utlandSjekker.add(BrukerManglerAdresse)
         }
 
-        val harFlyttetFraNorge = person.utland?.verdi?.utflyttingFraNorge?.isNotEmpty() ?: false
-        val harFlyttetTilNorge = person.utland?.verdi?.innflyttingTilNorge?.isNotEmpty() ?: false
-        if (harFlyttetTilNorge || harFlyttetFraNorge) {
-            utlandSjekker.add(SoekerHarBoddUtland)
```

Comment:
Er SoekerHarBoddUtland fjernet fordi det ikke gjelder for > 18  og vi er ferdig med resten?

Reply:
Nei, den ble splittet i to ulike sjekker for inn og utvandring :)